src/nginx/t/auth_pkey.t

```diff
@@ -113,7 +113,7 @@ foreach my $pkey($pkey_jwk, $pkey_x509) {
   like($response, qr/Content-Type: application\/json/i,
        'Missing creds returned application/json body.');
   like($response, qr/JWT validation failed: Missing or invalid credentials/i,
-       "Error body contains 'Missing or invalid credentials'.");
+                  "Error body contains 'Missing or invalid credentials'.");
```

Comment:
nit: indentation for `like` is not consistent.

Reply:
Done